cyclops-ui/src/components/pages/NewModule/NewModule.tsx

```diff
@@ -1283,18 +1284,24 @@ const NewModule = () => {
                   setLoadingValuesModal(true);
                 }}
                 name="Save"
+                disabled={loadingTemplate}
               >
                 Load values from file
               </Button>{" "}
               <Button
                 type="primary"
-                loading={loading}
+                loading={loading || loadingTemplate}
                 htmlType="submit"
                 name="Save"
+                disabled={loadingTemplate}
```

Comment:
Can you also disable the button if `loadingTemplateInitialValues` is set? This is set to true while Cyclops is fetching initial values from a template. You can do it for all buttons

Reply:
@ChiragAgg5k, you will have to add loadingTemplateInitialValues to disabled as well.